src/views/ipv/page/pyi-triage-select-smartphone.njk

```diff
@@ -0,0 +1,83 @@
+  <p class="govuk-body govuk-!-margin-bottom-2">{{ 'pages.pyiTriageSelectSmartphone.content.iphone.paragraph' | translate }}</p>
+  <ul class="govuk-list govuk-list--bullet ">
+    <li>{{ 'pages.pyiTriageSelectSmartphone.content.iphone.condition1' | translate }}</li>
+    <li>{{ 'pages.pyiTriageSelectSmartphone.content.iphone.condition2' | translate }}</li>
+  </ul>
+  {{ govukDetails({
+      summaryText: 'pages.pyiTriageSelectSmartphone.content.iphone.infoLabel' | translate,
+      text: 'pages.pyiTriageSelectSmartphone.content.iphone.infoHtml' | translate( { url: contactUsUrl } ) | safe
+  }) }}
+
+  <h2 class="govuk-heading-m govuk-!-margin-top-7">{{ 'pages.pyiTriageSelectSmartphone.content.android.subHeading' | translate }}</h2>
+  <p class="govuk-body govuk-!-margin-bottom-2">{{ 'pages.pyiTriageSelectSmartphone.content.android.paragraph' | translate }}</p>
+  <ul class="govuk-list govuk-list--bullet ">
+    <li>{{ 'pages.pyiTriageSelectSmartphone.content.android.condition1' | translate }}</li>
+    <li>{{ 'pages.pyiTriageSelectSmartphone.content.android.condition2' | translate }}</li>
+  </ul>
+  {{ govukDetails({
+    summaryText: 'pages.pyiTriageSelectSmartphone.content.android.infoLabel' | translate,
+    text: 'pages.pyiTriageSelectSmartphone.content.android.infoHtml' | translate( { url: contactUsUrl } ) | safe
+  }) }}
+
+  <h2 class="govuk-heading-m govuk-!-margin-top-7">{{ 'pages.pyiTriageSelectSmartphone.content.subHeadingAppendix' | translate }}</h2>
+  <p class="govuk-body">{{ 'pages.pyiTriageSelectSmartphone.content.paragraphAppendix1' | translate }}</p>
+  <p class="govuk-body">{{ 'pages.pyiTriageSelectSmartphone.content.paragraphAppendix2' | translate }}</p>
+
+  <form id="pyiTriageSelectSmartphoneOptionsForm" action="/ipv/page/{{ pageId }}" method="POST">
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+    {% set radiosConfig = {
+      idPrefix: "journey",
+      name: "journey",
+      fieldset: {
+        legend: {
+          text: 'pages.pyiTriageSelectSmartphone.content.subHeadingSelectSmartphone' | translate,
+          classes: "govuk-fieldset__legend--m"
+        }
+      },
+      items: [
+        {
+          value: "iphone",
+          text: 'pages.pyiTriageSelectSmartphone.content.formRadioButtons.iphoneButtonText' | translate
+        },
+        {
+          value: "android",
+          text: 'pages.pyiTriageSelectSmartphone.content.formRadioButtons.androidButtonText' | translate
+        },
+        {
+          value: "end",
+          text: 'pages.pyiTriageSelectSmartphone.content.formRadioButtons.neitherButtonText' | translate
+        }
+      ]
+    } %}
+
+    {% if errorState %}
+      {% set errorMessageObject = { 'text': 'pages.pyiTriageSelectSmartphone.content.formErrorMessage.errorRadioMessage' | translate } %}
+      {% set radiosConfig = radiosConfig | setAttribute('errorMessage', errorMessageObject) %}
+    {% endif %}
+
+    {{ govukRadios(radiosConfig) }}
+    {{ govukButton({
```

Comment:
I dont think this is the button we want here. We use this on pages where it is unrecoverable and direct the user to gov.uk

Reply:
Issue references [here](https://github.com/govuk-one-login/ipv-core-front/pull/1339#discussion_r1530165985)

Reply:
I think you'll need to add `id: submitButton` for the event listener which disables the button on click